imagery/i.landsat.toar/main.c

```diff
@@ -319,7 +319,8 @@ int main(int argc, char *argv[])
 	G_debug(1, "lsat.number = %d, lsat.sensor = [%s]",
 		lsat.number, lsat.sensor);
 
-	if (!lsat.sensor || lsat.number > 8 || lsat.number < 1)
+	/* address of array 'lsat.sensor' will always evaluate to 'true' */
+        if (!*lsat.sensor || lsat.number > 8 || lsat.number < 1)
```

Comment:
```suggestion
        if (!lsat.sensor[0] || lsat.number > 8 || lsat.number < 1)
```
lsat.sensor is an array `char sensor[10];`

Reply:
Updated as suggested.